projects/plugins/jetpack/_inc/client/my-plan/my-plan-header/index.js

```diff
@@ -117,11 +117,42 @@ class MyPlanHeader extends React.Component {
 				};
 
 			case 'is-security-t1-plan':
+				return {
+					...productProps,
+					details: expiration,
+					tagLine: createInterpolateElement(
+						sprintf(
+							/* translators: %1$d is the number of gigabytes of storage space the site has. */
+							_n(
+								'Enjoy the peace of mind of complete site protection. You have <strong>%1$dGB</strong> of storage space.',
+								'Enjoy the peace of mind of complete site protection. You have <strong>%1$dGB</strong> of storage space.',
+								TIER_1_BACKUP_STORAGE_GB,
+								'jetpack'
+							),
+							TIER_1_BACKUP_STORAGE_GB
+						),
+						{ strong: <strong /> }
+					),
+					title: __( 'Jetpack Security', 'jetpack' ),
+				};
+
 			case 'is-security-t2-plan':
 				return {
 					...productProps,
 					details: expiration,
-					tagLine: __( 'Enjoy the peace of mind of complete site protection.', 'jetpack' ),
+					tagLine: createInterpolateElement(
+						sprintf(
+							/* translators: %1$d is the number of gigabytes of storage space the site has. */
+							_n(
+								'Enjoy the peace of mind of complete site protection. You have <strong>%1$dTB</strong> of storage space.',
+								'Enjoy the peace of mind of complete site protection. You have <strong>%1$dTB</strong> of storage space.',
+								TIER_2_BACKUP_STORAGE_TB,
+								'jetpack'
+							),
```

Comment:
I might be misunderstanding something because I think we don't need to wrap everything with `_n()`. Both strings look the same regardless of the number of GB/TB the plan offers.

Reply:
I did this out of an abundance of caution, but I'm happy to change it if it's not gaining us anything. It's my understanding that in some languages, singular and plural forms change depending on the number, even beyond the case of 1 vs >1. So I wrote using `_n` in case, e.g., Language A considers 10 plural while Language B considers it singular.

@Automattic/i18n, can you lend any insight to how we should handle this sort of situation?

Reply:
TIL. I never thought before about languages having different singular and plural forms. 

I guess it won't hurt to keep those `_n()` calls, so I'm approving your changes.

Reply:
@mattgawarecki, that's correct. In this case it's preferred to keep the `_n()` call.